packages/data-point/lib/entity-types/base-entity/factory.js

```diff
@@ -12,10 +12,21 @@ function create (Factory, spec, id) {
   const entity = new Factory(spec)
 
   entity.id = id
+
+  if (spec.before) {
+    entity.before = createTransform(spec.before)
+  }
+
   entity.value = createTransform(spec.value)
```

Comment:
we shouldn't add the value property if it's falsey

Reply:
Done. This did not require changes to `base-entity/resolve`.